app/consumer.py

```diff
@@ -60,6 +61,7 @@ def on_message(self, unused_channel, basic_deliver, properties, body):
 
 def main():
     logger.debug("Starting consumer")
+    logger.info("Current version: {}".format(__version__))
```

Comment:
Good point - should have checked Splunk's ingesting preferences first! I'll update this and other sdx-* services accordingly.